parts/k8s/cloud-init/artifacts/chrony.conf

```diff
@@ -0,0 +1,43 @@
+# Use Microsoft NTP servers:
```

Comment:
@cpuguy83 @khenidak Can you kindly comment on a good, standard, robust Azure-proximate NTP config? We should probably sync to more URIs than just `time.windows.com`, right?

(Also feel free to inspect the config below generally for operational hygiene.)